dev/Expander/Expander.xaml

```diff
@@ -23,14 +23,13 @@
         <Setter Property="Background" Value="{ThemeResource ExpanderBackground}" />
         <contract7Present:Setter Property="BackgroundSizing" Value="InnerBorderEdge" />
         <Setter Property="MinWidth" Value="{ThemeResource FlyoutThemeMinWidth}" />
-        <Setter Property="MaxWidth" Value="{ThemeResource FlyoutThemeMaxWidth}" />
         <Setter Property="MinHeight" Value="{StaticResource ExpanderMinHeight}" />
         <Setter Property="BorderThickness" Value="{ThemeResource ExpanderBorderThickness}" />
         <Setter Property="BorderBrush" Value="{ThemeResource ExpanderBorderBrush}" />
         <Setter Property="Padding" Value="{StaticResource ExpanderHeaderPadding}" />
-        <Setter Property="HorizontalAlignment" Value="Stretch" />
-        <Setter Property="HorizontalContentAlignment" Value="Left" />
-        <Setter Property="VerticalContentAlignment" Value="Center" />
+        <Setter Property="HorizontalAlignment" Value="Left" />
+        <Setter Property="HorizontalContentAlignment" Value="Stretch" />
+        <Setter Property="VerticalContentAlignment" Value="Stretch" />
```

Comment:
this needs to be center

Reply:
I don't know which property you are talking about specifically. Since HorizontalAlignment=Left is the standard for all controls, I'll assume ContentAlignment. ContentAlignment in this case Must be stretched I beleive. Again, this control needs to be responsive to size either to the layout root grid's constraints or the content size within the ContentPresenter.

Reply:
Reviewing Button's template, it does not set these properties. This means the default is used which is in fact `Center`. I'll go ahead and delete the properties entirely.

Reply:
Deleted these and then added VerticalAlignment=Center to follow Button. However, ComboBox has `<Setter Property="VerticalAlignment" Value="Top" />` so WinUI is inconsistent here.

Reply:
Vertical alignment does need to be center.